src/App.js

```diff
@@ -9,23 +9,41 @@ export default class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies
+      movies: [],
+      error: ''
     }
   }
-
-  displayMovieDetails = (id) => {
-    const foundMovie = this.state.movies.filter(movie => movie.id === id);
-    this.setState({movies: foundMovie})
+  
+  componentDidMount = async() => {
+    let fetchedMovies = [];
```

Comment:
This seems like extra work. The variable could be scoped inside that try block, because nothing outside of it needs to access `fetchedMovies`.